src/App.js

```diff
@@ -0,0 +1,18 @@
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import Book from './components/Book';
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <>
+      <Navbar />
+      <Routes>
+        <Route path="/" elemen={<Book />} t />
+        <Route path="/categories" elemen={<Book />} t />
```

Comment:
- [ ] Kindly fix this route, it should have a different from `Book`, since this redirect to `/categories`